logos/src/logos/dbutils/dbmanager.py

```
@@ -3236,6 +3236,139 @@ def user_authorization(self, logos_key: str):
+
+        return {
+            "id": user_id,
+            "username": username,
+            "prename": prename,
+            "name": name,
+            "email": email,
+            "role": role,
+            "teams": [],
+        }, logos_key, 200
+
+    def delete_user(self, user_id: int) -> tuple[dict, int]:
```

Flake8 (7.3.0)

[error] redefinition of unused 'delete_user' from line 323

(F811)